src/components/forms/PractitionerForm/UserIdSelect/tests/index.test.tsx

```diff
@@ -12,6 +12,7 @@ import { practitioners, sortedUsers, users } from './fixtures';
 // tslint:disable-next-line: no-var-requires
 const fetch = require('jest-fetch-mock');
 jest.mock('../../../../../configs/env.ts', () => ({
+  PRACTITIONER_REQUEST_PAGE_SIZE: 1000,
```

Comment:
Should we raise the default for practitioners to be more than 1000?  We want to strike a balance between the number of requests we make, and the size of the response we receive from OpenSRP. I think if we can figure out what page length results in about 100Kb of practitioners then we should use that page size. The figure of 100Kb is loosely based on [this discussion](https://www.postgresql.org/message-id/4D2F2C71.8080805%40dndg.it) - not scientific.

Reply:
Just to make it clear, not suggesting that you try and determine the number of practitioners in 100Kb programmatically.  Just do some manual testing with the API.

Reply:
1000 records are more than 100kb. 100kb is about 650 records